gratipay/models/community.py

```diff
@@ -3,14 +3,14 @@
 from postgres.orm import Model
 
 
-name_pattern = re.compile(r'^[A-Za-z0-9,._ -]+$')
+name_pattern = re.compile(r"^[A-Za-z0-9,._ '-]+$")
```

Comment:
I don't think we should modify the semantics of the code we have in production. I think instead we should modify fake data to not use community names that don't pass `name_pattern`.